packages/desktop-client/src/components/tutorial/Final.js

```diff
@@ -10,8 +10,12 @@ function Final({ targetRect, navigationProps }) {
       <Title>That's it!</Title>
       <P>
         With that workflow you can have peace of mind that what you are looking
-        at reflects reality. &#128524; The amount of money in a category is cash
-        that you can safely spend right now.
+        at reflects reality.{' '}
+        <span role="img" aria-label="Relieved smile">
```

Comment:
IMO the emoji here provide only a small amount of value — maybe make them `aria-hidden` so screen reader users can get to the point more quickly? (Same on the other changed file below)

Reply:
Makes sense https://github.com/actualbudget/actual/pull/415/commits/9c1d68c18c08e7af7f50798f02ac7214bbfaec84